packages/liveblocks-node/src/webhooks.ts

```diff
@@ -58,11 +59,18 @@ export class WebhookHandler {
   /**
    * Verifies the headers and returns the webhookId, timestamp and rawSignatures
    */
-  private verifyHeaders(headers: IncomingHttpHeaders) {
+  private verifyHeaders(headers: IncomingHttpHeaders | Headers) {
     const sanitizedHeaders: IncomingHttpHeaders = {};
-    Object.keys(headers).forEach((key) => {
-      sanitizedHeaders[key.toLowerCase()] = headers[key];
-    });
+
+    if (headers instanceof Headers) {
```

Comment:
Not sure `instanceof Headers` works in a node environment. I don't think `Headers` is globally available in node. We should probably add an additional check.

Reply:
Good shout, `Headers` have only been supported in Node.js since `18.0.0`. Added a check.

Reply:
Nice, also feel free to add an example with `Headers` in the `WebhookRequest` type if you think it helps!

Reply:
@GuillaumeSalles they do but on the off chance, I'd rather be safe

Reply:
Done @FlowFlorent!